### ISSUE TITLE
1.4.5 item deletions

### ISSUE THREAD

**Issue by ImTheStigHonest** [scalda/FoolCraft#560]
## SUGGESTION/HELP
- [ ] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

## Issue Description
avaritia neutron collectors vanished and neutron piles etc in compacting drawers have been changed to high end items see screen shot
storage drawers- several storage drawers deleted (custom framed only)
nesting pens became entities, no longer where placed

## Reproduce Steps
first load with 1.4.5

## Additional Information
http://imgur.com/4x2vjAz

**Comment by Jo-Mamma75** [comment]
The missing items were probably from Avaritia Additions which is no longer in the pack so that would likely explain the first item on the list as well as possibly explaining the second item on the list.  As for framed drawers going missing, the only thing I can think of is that the material used to make the drawers has either been removed by the mod author or something got changed with the item ID by the mod author.  As for the nesting pens, I couldn't tell you other than it too was likely due to an update by the mod author.

My suggestion would be to revert to the previous version and the backup I'm sure that you made prior to updating.  Make note of everything that you now have in the previous version that was missing in the updated version.  Take all your chickens out of their nesting pens and place them into animal nets or golden lassos.  Move the items from the framed drawers into other drawers.  Update the pack again.  Cheat in the missing items from Avaritia and attempt to craft the same framed drawers again to see if they work.

**Comment by ImTheStigHonest** [comment]
Thanks for the update. just for info and this is strange. I had 7 drawers all the same only 6 were deleted.
I had already taken the chickens out as I am still experiencing the ticking entities crash with some chickens and have yet to determine exactly which were causing the crash

**Comment by Jo-Mamma75** [comment]
We had another player that reported issues with the chickens but we tried to reproduce it on a world download he provided and could not.  We then updated a backup of 3 different servers, all of which had tons of chickens in them, and still could not reproduce the problem.  Not that I don't believe you have the issue.  It's just difficult to troubleshoot from our end without being able to reproduce it and work it to resolution.  The only thing I can suggest is to make note of the chickens you have, kill them all, update and cheat in eggs for those specific chickens and breed them up again.
